packages/better-auth/src/db/internal-adapter.ts

```
@@ -536,15 +536,19 @@ export const createInternalAdapter = (
 			const account = await adapter
```

If the account ID isn't defined, we should probably just return an error. For providers that don't return an ID, we might need to fall back to the user ID or another consistent, unique identifier